src/main/java/browserview/BrowserComponent.java

```diff
@@ -123,12 +130,25 @@ private ChromeDriverEx createChromeDriver() {
         ChromeDriverEx driver = new ChromeDriverEx(options, isTestChromeDriver);
         WebDriver.Options manage = driver.manage();
         if (!isTestChromeDriver) {
+            setWindowSize(manage);
+            initialiseJNA();
+        }
+        return driver;
+    }
+
+    public void setWindowSize(WebDriver.Options manage) {
```

Comment:
Does it make sense to make this a static method and then get `UI` passed in explicitly as a parameter?